src/ctap/mod.rs

```diff
@@ -240,16 +238,12 @@ where
         let mut iv = [0; 16];
         iv.copy_from_slice(&self.rng.gen_uniform_u8x32()[..16]);
 
-        let block_len = if cred_random.is_some() { 6 } else { 4 };
+        let block_len = 4;
```

Comment:
Maybe this `block_len` is not needed anymore now that all the sizes are known at compile time.

Reply:
Both `block_len` removed.